Dockerfile

```diff
@@ -1,8 +1,15 @@
-FROM node:10.17-alpine
+FROM docker.io/library/node:14-alpine
 
```

Comment:
we should add an .nvmrc file in the repository too

Reply:
Is added. Do we need more changes if we want use node 14 for production? @Loki-Afro

Reply:
could someone explain me why you think that .nvmrc is needed at build time? i have seen that in a dockerfile but it is not using nvm at all